### PR TITLE
Remove bad references (dead links)

### PR DESCRIPTION
**Pull request by wchen-r7**
These links are no longer available (404). They are dead links.

These bad links were originally picked up by tools/modules/module_reference.rb, and then manually verified one by one while I was updating the modules.
- [x] To verify, you can manually try the links.

### CODE REVIEW

File: modules/auxiliary/admin/http/arris_motorola_surfboard_backdoor_xss.rb [hunk 1]
```diff
@@ -46,8 +46,7 @@ def initialize(info = {})
       'References' => [
         [ 'CVE', '2015-0964' ], # XSS vulnerability
         [ 'CVE', '2015-0965' ], # CSRF vulnerability
-        [ 'CVE', '2015-0966' ],  # "techician/yZgO8Bvj" web interface backdoor
-        [ 'URL', 'https://community.rapid7.com/rapid7_blogpostdetail?id=a111400000AanBs' ] # Original disclosure
```
Comment (jlee-r7) [inline modules/auxiliary/admin/http/arris_motorola_surfboard_backdoor_xss.rb]: https://community.rapid7.com/community/infosec/blog/2015/06/05/r7-2015-01-csrf-backdoor-and-persistent-xss-on-arris-motorola-cable-modems

File: modules/auxiliary/admin/http/manage_engine_dc_create_admin.rb [hunk 2]
```diff
@@ -27,7 +27,6 @@ def initialize(info = {})
         [
           ['CVE', '2014-7862'],
           ['OSVDB', '116554'],
-          ['URL', 'https://raw.githubusercontent.com/pedrib/PoC/master/ManageEngine/me_dc9_admin.txt'],
```
Comment (jlee-r7) [inline modules/auxiliary/admin/http/manage_engine_dc_create_admin.rb]: https://github.com/pedrib/PoC/blob/master/advisories/ManageEngine/me_dc9_admin.txt

File: modules/auxiliary/admin/http/manageengine_dir_listing.rb [hunk 3]
```diff
@@ -36,7 +36,6 @@ def initialize(info={})
         [
           ['CVE', '2014-7863'],
           ['OSVDB', '117696'],
-          ['URL', 'https://raw.githubusercontent.com/pedrib/PoC/master/ManageEngine/me_failservlet.txt'],
```
Comment (jlee-r7) [inline modules/auxiliary/admin/http/manageengine_dir_listing.rb]: https://github.com/pedrib/PoC/blob/master/advisories/ManageEngine/me_failservlet.txt

File: modules/auxiliary/admin/http/manageengine_file_download.rb [hunk 4]
```diff
@@ -34,7 +34,6 @@ module will attempt to login using the default credentials for the administrator
         [
           ['CVE', '2014-7863'],
           ['OSVDB', '117695'],
-          ['URL', 'https://raw.githubusercontent.com/pedrib/PoC/master/ManageEngine/me_failservlet.txt'],
```
Comment (jlee-r7) [inline modules/auxiliary/admin/http/manageengine_file_download.rb]: https://github.com/pedrib/PoC/blob/master/advisories/ManageEngine/me_failservlet.txt

File: modules/auxiliary/admin/http/manageengine_pmp_privesc.rb [hunk 5]
```diff
@@ -34,7 +34,6 @@ def initialize(info = {})
         [
           [ 'CVE', '2014-8499' ],
           [ 'OSVDB', '114485' ],
-          [ 'URL', 'https://raw.githubusercontent.com/pedrib/PoC/master/ManageEngine/me_pmp_privesc.txt' ],
```
Comment (jlee-r7) [inline modules/auxiliary/admin/http/manageengine_pmp_privesc.rb]: https://github.com/pedrib/PoC/blob/master/advisories/ManageEngine/me_pmp_privesc.txt

File: modules/auxiliary/admin/http/netflow_file_download.rb [hunk 6]
```diff
@@ -28,7 +28,6 @@ def initialize(info={})
         [
           [ 'CVE', '2014-5445' ],
           [ 'OSVDB', '115340' ],
-          [ 'URL', 'https://raw.githubusercontent.com/pedrib/PoC/master/ManageEngine/me_netflow_it360_file_dl.txt' ],
```
Comment (jlee-r7) [inline modules/auxiliary/admin/http/netflow_file_download.rb]: https://github.com/pedrib/PoC/blob/master/advisories/ManageEngine/me_netflow_it360_file_dl.txt

File: modules/auxiliary/admin/http/sysaid_admin_acct.rb [hunk 7]
```diff
@@ -27,7 +27,6 @@ def initialize(info = {})
       'References' =>
         [
           [ 'CVE', '2015-2993' ],
-          [ 'URL', 'https://raw.githubusercontent.com/pedrib/PoC/master/generic/sysaid-14.4-multiple-vulns.txt' ],
```
Comment (jlee-r7) [inline modules/auxiliary/admin/http/sysaid_admin_acct.rb]: https://github.com/pedrib/PoC/blob/master/advisories/sysaid-14.4-multiple-vulns.txt

File: modules/auxiliary/admin/http/sysaid_file_download.rb [hunk 8]
```diff
@@ -34,7 +34,6 @@ def initialize(info={})
         [
           ['CVE', '2015-2996'],
           ['CVE', '2015-2997'],
-          ['URL', 'https://raw.githubusercontent.com/pedrib/PoC/master/generic/sysaid-14.4-multiple-vulns.txt'],
```
Comment (jlee-r7) [inline modules/auxiliary/admin/http/sysaid_file_download.rb]: https://github.com/pedrib/PoC/blob/master/advisories/sysaid-14.4-multiple-vulns.txt

File: modules/auxiliary/admin/http/sysaid_sql_creds.rb [hunk 9]
```diff
@@ -29,7 +29,6 @@ def initialize(info={})
         [
           ['CVE', '2015-2996'],
           ['CVE', '2015-2998'],
-          ['URL', 'https://raw.githubusercontent.com/pedrib/PoC/master/generic/sysaid-14.4-multiple-vulns.txt' ],
```
Comment (jlee-r7) [inline modules/auxiliary/admin/http/sysaid_sql_creds.rb]: https://github.com/pedrib/PoC/blob/master/advisories/sysaid-14.4-multiple-vulns.txt

File: modules/auxiliary/bnat/bnat_router.rb [hunk 10]
```diff
@@ -21,8 +21,7 @@ def initialize
       'License'      => MSF_LICENSE,
       'References'   =>
         [
-          [ 'URL', 'https://github.com/claudijd/BNAT-Suite'],
```
Comment (jlee-r7) [inline modules/auxiliary/bnat/bnat_router.rb]: https://github.com/claudijd/bnat

File: modules/auxiliary/bnat/bnat_scan.rb [hunk 11]
```diff
@@ -25,8 +25,7 @@ def initialize
       'License'      => MSF_LICENSE,
       'References'   =>
         [
-          [ 'URL', 'https://github.com/claudijd/BNAT-Suite'],
```
Comment (jlee-r7) [inline modules/auxiliary/bnat/bnat_scan.rb]: https://github.com/claudijd/bnat

File: modules/auxiliary/dos/ssl/openssl_aesni.rb [hunk 12]
```diff
@@ -28,8 +28,7 @@ def initialize(info = {})
       'License'		=> MSF_LICENSE,
       'References'	=>
         [
-          [ 'CVE', '2012-2686'],
-          [ 'URL', 'https://www.openssl.org/news/secadv_20130205.txt']
```
Comment (jlee-r7) [inline modules/auxiliary/dos/ssl/openssl_aesni.rb]: https://www.openssl.org/news/secadv/20130205.txt

**Comment by wchen-r7** [comment]
Ok, cool I will update them. Thanks!

**Comment by jlee-r7** [comment]
For the record, I much prefer to see commas after the last element of Arrays so diffs like this aren't so big and are easier to read.

**Comment by wchen-r7** [comment]
All links mentioned above are updated. Thanks.

**Pull request merged**